### PR TITLE
[8.0] Port l10n_ca_qc_account_fiscal_position_rule module

### PR DESCRIPTION
**Pull request by Ehtaga**
Depends on
- [x] account_fiscal_position_rule_sale https://github.com/OCA/account-fiscal-rule/pull/25
- [x] account_fiscal_position_rule_purchase https://github.com/OCA/account-fiscal-rule/pull/26
- [x] account_fiscal_position_rule_stock https://github.com/OCA/account-fiscal-rule/pull/30
- [x] l10n_ca_toponyms
- [x] l10n_ca

### CODE REVIEW

File: l10n_ca_qc_account_fiscal_position_rule/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+* ...
+
```
Comment (bwrsandman) [inline l10n_ca_qc_account_fiscal_position_rule/README.rst:21]: Take out empty sections

**Comment by max3903** [comment]
@Ehtaga When I installed the module, the rules are imported but the field `fiscal_position_id` is empty in all rules.

**Comment by max3903** [comment]
:+1:

**Comment by plamarche** [comment]
:+1:

**Pull request merged**